### ISSUE TITLE
[Ubuntu] Context menu blank ?

### ISSUE THREAD

**Issue by Silvest89** [warpdotdev/Warp#4387]
### Describe the bug

Please see the below image

### To reproduce

Right click on a selection to open the context menu

### Screenshots

![image](https://github.com/warpdotdev/Warp/assets/2388338/60392407-ee16-4387-835a-d4087fdcfd5f)

### Operating system

Linux

### Operating system and version

Ubuntu 23.10

### Shell Version

zsh 5.9

### Current Warp version

v0.2024.02.20.08.01.stable_02

### Regression

No, this bug or issue has existed throughout my experience using Warp

### Does this block you from using Warp daily?

No

### Is this a Warp specific issue? (i.e. does it happen in Terminal, iTerm, Kitty, etc.)

Yes, this I confirmed this only happens in Warp, not other terminals.

### Warp Internal (ignore): linear-label:b9d78064-c89e-4973-b153-5178a31ee54e

None

**Comment by vorporeal** [comment]
Very strange! Does this reproduce consistently for you? Do any other parts of the app appear with missing text?

**Comment by alokedesai** [comment]
@Silvest89 Friendly ping on this. Additionally, do you see anything in the log output of `warp-terminal` when this issues occurs?

**Comment by Silvest89** [comment]
@alokedesai I found out that my ubuntu installation by default misses a couple of fonts Hack, DejaVu amonst others. I switched to FreeMono and everything is visible again. Also in Settings=>Appearance

**Comment by alokedesai** [comment]
@Silvest89 Just to confirm, are you saying that changing the font in `Settings > Appearance` from Hack to `FreeMono` solved the issue? 

If so, would you mind changing the font back to Hack and send the output of your logs so we can debug further? Thank you!

**Comment by Silvest89** [comment]
@alokedesai 
Yes it did. :) The set of fonts hack, dejavu etc are not installed by default on Ubuntu. See the below video. The latest update fixed the Appearance screen (it had the same bug as the terminal window not showing text.)
If you look at the Prompt section you can see that the settings screen supports the majority of the fonts. But the terminal window doesn't.

[Screencast from 2024-03-02 03-44-41.webm](https://github.com/warpdotdev/Warp/assets/2388338/a7a7ab39-26ba-42eb-9896-4acd40aebb1a)
